backend/dal/Services/LeaseService.cs

```diff
@@ -120,6 +120,13 @@ public Lease Get(int id)
                     .ThenInclude(i => i.BctfaRiskManagementContact)
                     .ThenInclude(p => p.ContactMethods)
 
+                .Include(l => l.SecurityDeposits)
+                .ThenInclude(s => s.SecurityDepositHolderType)
```

Comment:
nit: since the ThenInclude corresponds to the previous statement we could tab them to make it easier to follow

Reply:
I assumed auto-format on save would take care but seems it won't. Sure I can indent it

Reply:
I dont think is smart enough to know that the chaining is working on different layers of the object :(